src/main/scala/li/cil/oc/client/renderer/font/StaticFontRenderer.scala

```diff
@@ -16,7 +21,7 @@ import scala.io.Source
  */
 class StaticFontRenderer extends TextureFontRenderer {
```

Comment:
I recommend to drop this functionality right here. It's a pointless maintenance burden which I planned I swear nobody really uses it anymore - it's all the Unifont .HEX file.

Reply:
There's an old VBO rewrite here for 1.7, but the code is not very good. Either way, it might give you some inspiration: https://github.com/MightyPirates/OpenComputers/pull/3138

Reply:
I'll probably remove it in the same pass as converting to the dedicated manual mod then, it appears to function for now.
I've already implemented a buffered approach for screens using MC's vertex buffers to properly support the `MatrixStack`.

Reply:
Excellent!